drivers/ukps2/kbd/Config.uk

```diff
@@ -0,0 +1,3 @@
+config LIBUKPS2_KBD
+	bool "PS/2 Keyboard dumbed down driver"
+	depends on ARCH_X86_64
```

Comment:
This needs `depends on HAVE_PS2` so that it's not visible on architectures that don't have that capability.